src/liballoc/boxed.rs

```diff
@@ -48,7 +48,7 @@
 //!
 //! It wouldn't work. This is because the size of a `List` depends on how many
 //! elements are in the list, and so we don't know how much memory to allocate
-//! for a `Cons`. By introducing a `Box`, which has a defined size, we know how
+//! for a `Cons`. By introducing a [`Box`], which has a defined size, we know how
```

Comment:
Should be `Box<T>` here?

Reply:
No reason to add the "<T>". Or maybe I missed something?

Reply:
Well it's the convention to use `<T>`; https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#referring-to-types

Reply:
Ah then I'll add it. :)